boltdesignsystem/bolt
# Bolt version generator

## Jira

Without JIRA ticket

## Summary

Fix some logic and add an updated version of the `.incache` file.

## Details

For every release, we should use the correct way to push the changes or if we create only `tag` in GitHub we should update the `.incache` file to have all versions available in our JSON file. We can achieve that by running those commands:
```
npx json -I -f docs-site/.incache -e 'this["bolt-tags"].expiresOn = "2019-06-14T12:30:26.377Z"'
npx json -I -f docs-site/.incache -e 'this["bolt-urls-to-test"].expiresOn = "2019-06-14T12:30:26.377Z"'
yarn setup (or) yarn start
```
I can run `npx now alias boltdesignsystem.com ${tagSpecificUrl} --token=${NOW_TOKEN}` because I have this error after running the command:
```
 Error! You do not have access to the specified account
```
I assume that my `NOW_TOKEN` doesn't have access to Bolt NOW.SH account. @danielamorse maybe you will have more luck on this.

When we create an alias URL for version `2.27.0` we should update the `.incache` file using the commands above.

## How to test

Check if after running this code locally you see `2.26.4` in the bolt version dropdown.
![image](https://user-images.githubusercontent.com/16049049/95197012-34cbc800-07d9-11eb-9a8a-506d162e63b2.png)
